wireapp/wire-ios-mono
# fix: conversation filter use case must filter the self user - WPB-11997

<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-11997" title="WPB-11997" target="_blank"><img alt="Bug" src="https://wearezeta.atlassian.net/rest/api/2/universal_avatar/view/type/issuetype/avatar/10803?size=medium" />WPB-11997</a>  [iOS] Searching/filtering conversations must not match against the self user
  </td></table>
  <br />
 





### Issue

The conversation filter use case currently considers all participants of a conversation. As a result too many conversations match the search text.
This PR filters the self user before the use case is invoked.

### Testing

- Tap the search field at the top of the conversation list.
- Enter the name of the self user.
- Ensure the result is correct.

---

### Checklist


---

### UI accessibility checklist

_If your PR includes UI changes, please utilize this checklist:_
